dns_check/tests/test_e2e.py

```diff
@@ -0,0 +1,15 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from copy import deepcopy
+
+import pytest
+
+from . import common
+
+
+@pytest.mark.e2e
+def test_check(dd_agent_check):
+    aggregator = dd_agent_check(deepcopy(common.INSTANCE_INTEGRATION))
+
+    aggregator.assert_metric('dns.response_time')
```

Comment:
+ `aggregator.assert_all_metrics_covered()` ?

Same for https://github.com/DataDog/integrations-core/blob/73a6d7e775a4f9c24cb7e7805313046a66610fb1/dns_check/tests/test_integration.py#L14-L17 ?

Reply:
yeah.... this is literally the only metric of this check